docs/manuals/run-deploy/deploy/index.md

```diff
@@ -16,8 +14,6 @@ Here are the different situations that Taipy can address:
   [Databricks](https://www.databricks.com/) platform;
 
 
-## [Taipy Cloud](taipy-cloud/index.md)
-
 ## [Windows](windows/index.md)
```

Comment:
@FabienLelaquais @FlorianJacta @florian-vuillemot 
I propose to remove all the titles here since they are already listed above with a short description. What do you think?